source/common/grpc/grpc_web_filter.cc

```diff
@@ -130,5 +168,47 @@ Http::FilterTrailersStatus GrpcWebFilter::encodeTrailers(Http::HeaderMap& traile
   return Http::FilterTrailersStatus::Continue;
 }
 
+void GrpcWebFilter::setupStatTracking(const Http::HeaderMap& headers) {
```

Comment:
This code is basically copied from https://github.com/lyft/envoy/blob/master/source/common/grpc/http1_bridge_filter.cc#L102 can we share?